go.mod

```diff
@@ -625,3 +625,5 @@ replace (
 	k8s.io/metrics => k8s.io/metrics v0.23.15
 	sigs.k8s.io/custom-metrics-apiserver => sigs.k8s.io/custom-metrics-apiserver v1.23.0
 )
+
+replace github.com/netsampler/goflow2 => github.com/AlexandreYang/goflow2 v1.0.5-0.20230421152336-42ada9facd76
```

Comment:
note: this is temporary, will be updated later once https://github.com/netsampler/goflow2/pull/161 is merged